aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -1217,35 +1221,59 @@ static Tensor& std_var_out(Tensor& result, const Tensor& self, IntArrayRef dim,
     ScalarType dtype = c10::toValueType(get_dtype_from_result(result, {}));
     Tensor real_in = at::real(self);
     Tensor real_out = at::empty({0}, self.options().dtype(dtype));
-    auto iter = make_reduction("std or var", real_out, real_in, dim, keepdim, dtype);
-    if (iter.numel() == 0) {
-      real_out.fill_(NAN);
-    } else {
-      std_var_stub(iter.device_type(), iter, unbiased, false);
-    }
+    std_var_out(
```

Comment:
Would you add a comment before this block explaining the std + var computation for complex so readers know what to expect